inst/mappings/mapping_ECEMF.csv

```diff
@@ -83,6 +233,7 @@ Emissions|CO2|Energy|Demand|Other Sector;Mt CO2/yr;Emi|CO2|Energy|Demand|+|CDR S
 Emissions|CO2|Energy|Demand|Residential;Mt CO2/yr;;;;;;;
 Emissions|CO2|Energy|Demand|Residential and Commercial;Mt CO2/yr;Emi|CO2|Energy|Demand|+|Buildings;Mt CO2/yr;1;;;;R
 Emissions|CO2|Energy|Demand|Transportation;Mt CO2/yr;Emi|CO2|w/o Bunkers|Energy|Demand|Transport;Mt CO2/yr;1;;;;R
+Emissions|CO2|Energy|Demand|Transportation (w/ bunkers);Mt CO2/yr;Emi|CO2|w/o Bunkers|Energy|Demand|Transport;Mt CO2/yr;1;;;;R
```

Comment:
is this correct? it looks like it maps "w/o bunkers" on "(w/ bunkers)"?

Reply:
The `renameOldInMappings()` piam interfaces function did this change and it is indeed wrong.

I am fixing this and I am trying to see if I am able to create a mapping that can support both REMIND 3.4 and 3.5 variable conventions at the same time.

I will put this merge in hold until I am able to do it.

Reply:
I adapted the mapping to fix the above mentioned issue and to support REMIND 3.5 variables convention.